src/Native/System.Native/pal_networking.h

```diff
@@ -84,3 +84,24 @@ extern "C" int32_t GetNameInfo(const uint8_t* address,
                                int32_t flags);
 
 extern "C" int32_t GetHostName(uint8_t* name, int32_t nameLength);
+
+struct LinkLayerAddressInfo
+{
+    uint32_t InterfaceIndex; // The index of the interface to which this address belongs.
+    uint8_t AddressBytes[8]; // A pointer to the bytes containing the address.
+    uint8_t NumAddressBytes; // The number of bytes actually stored in the address.
+    uint8_t __padding;
```

Comment:
What is the best way to do this? I did this because of the code above in the file:
````C++
struct PalIpAddress
{
    uint8_t* Address;    // Buffer to fit IPv4 or IPv6 address
    int32_t Count;       // Number of bytes in the address buffer
    bool IsIpv6;         // If this is an IPv6 Address or IPv4
    uint8_t reserved[3]; // Padding on 64 bit systems to align struct and not compile with a warning.
};
````